extensions/amp-analytics/0.1/variables.js

```diff
@@ -309,67 +309,68 @@ export class VariableService {
       }
 
       let value = options.getVar(name);
+      const bindings = opt_bindings || this.getMacros(element);
+      const urlReplacements = Services.urlReplacementsForDoc(element);
 
       if (typeof value == 'string') {
-        value = this.expandValue_(
+        value = this.expandValueAndReplaceAsync_(
           value,
           options,
           element,
+          bindings,
+          urlReplacements,
           opt_bindings,
-          opt_whitelist
+          opt_whitelist,
+          argList
         );
       } else if (isArray(value)) {
         // Treat each value as a template and expand
         for (let i = 0; i < value.length; i++) {
           value[i] =
             typeof value[i] == 'string'
-              ? this.expandValue_(
+              ? this.expandValueAndReplaceAsync_(
                   value[i],
                   options,
                   element,
+                  bindings,
```

Comment:
`bingdings` and `opt_bindings` : )
I don't see the need to pass `bindings` and `urlReplacements`